file-formats/chdo/examples/z_aff_example_chdo.chdo.json

```
@@ -38,6 +38,6 @@
     "number": "600"
   },
   "generatedObject": {
-    "classOrFunction": ""
+    "name": ""
```

@wurzka is it allowed to have an empty entry? I mean from an ABAP perspective. (JSON Schema wise, it is fine!)
The deserialized object will have errors, right? The mandatory field is initial, so this a syntax error I guess.

---

Mh, difficult to say. This entry is read-only and gets created by activating. So I'm not sure if it would result in a syntax error.
Maybe we should fill it with something like 
```suggestion
    "name": "ZCL_AFF_EXAMPLE_CHDO"
```

---

I see, the activation process will hopefully provide the information.